### DISCUSSION TITLE
Ability to customize User Tools menu

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5659] [Feature Requests]
Asked during the accessibility webinar.
Users want the ability to hide particular user tools. Other customizations would be nice too.

**Comment by maxpatiiuk** [comment]
Some user-tools customization was added in https://github.com/specify/specify7/pull/2725, but not much (if item is hidden from header, it appears in user tools)

**Comment by grantfitzsimmons** [comment]
We can already control what displays in the navigation menu and whatever does not appears in the user tools menu. 

This could be done by adding a third column in preferences for hidden items, or by adding a new section where you can select items to hide.
